components/LevitatingLink.js

```diff
@@ -1,7 +1,8 @@
-export default function LevitatingLink ({ href, children }) {
+export default function LevitatingLink ({ href, children, handleClick, active }) {
```

Comment:
Thank for your advice.Can in use getElementByID instead Refs.How do you think about this point?